pkg/cmd/import.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/G-Research/fasttrackml/pkg/database"
+)
+
+var ImportCmd = &cobra.Command{
+	Use:   "import",
+	Short: "Copies an input database to an output database",
+	Long: `The import command will transfer the contents of the input
+         database to the output database. Please make sure that the
+         FasttrackML server is not currently connected to the input
+         database.`,
+	RunE: importCmd,
+}
+
+func importCmd(cmd *cobra.Command, args []string) error {
+	// 1. init database connections.
+	inputDB, outputDB, err := initDBs()
+	if err != nil {
+		return err
+	}
+	defer inputDB.Close()
+	defer outputDB.Close()
+
+	dryRun := viper.GetBool("dry-run")
+	if err := database.Import(inputDB, outputDB, dryRun); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// initDBs inits the input and output DB connections.
+func initDBs() (input, output *database.DbInstance, err error) {
+	databaseSlowThreshold := time.Second * 1
+	databasePoolMax := 20
+	databaseReset := false
+	databaseMigrate := true
```

Comment:
we need to avoid migrating the input DB — ideally we even enforce some form of read-only functionality on its connection (if that's possible?)

Reply:
okay will do, i was just thinking this would ensure they are both at the right version automatically

Reply:
ah right, I forgot this was our MVP where we require the migrations to be done!
then maybe the migration should be behind a flag that is `false` by default?

in the future (_not_ in this PR), we will want to ensure the input DB is read-only and that we can deal with older schemas